testfiles/RadLoHydrHeatCoolAuto.idf

```diff
@@ -1951,19 +1951,19 @@
     0.013,                   !- Hydronic Tubing Inside Diameter {m}
     autosize,                !- Hydronic Tubing Length {m}
     MeanAirTemperature,      !- Temperature Control Type
-    HeatingDesignCapacity,   !- Heating Design Capacity Method
-    Autosize,                !- Heating Design Capacity {W}
+    FractionOfAutosizedHeatingCapacity,   !- Heating Design Capacity Method
+    ,                        !- Heating Design Capacity {W}
     ,                        !- Heating Design Capacity Per Floor Area {W/m2}
-    ,                        !- Fraction of Autosized Heating Design Capacity
+    1.0,                     !- Fraction of Autosized Heating Design Capacity
```

Comment:
It might be good to illustrate something other than a fraction of 1.0 since that is effectively the same as using the "HeatingDesignCapacity" with autosize. Is this value really a fraction, or is it a multiplier? i.e., can it be greater than 1.0?